.github/workflows/acceptance-test.yml

```diff
@@ -0,0 +1,62 @@
+#
+# This GitHub action runs acceptance tests every night.
+#
+name: "Acceptance Test"
+
+on:
+  schedule:
+    # Runs against the default branch every day at midnight
+    - cron: "0 0 * * *"
+  workflow_dispatch:
```

Comment:
This was added for testing purposes. I would like to run the action manually to ensure all is working as expected.